### PR TITLE
Add functions to get/set baseline

### PR DESCRIPTION
**Pull request by anlorn**

### CODE REVIEW

File: adafruit_ccs811.py [hunk 1]
```diff
@@ -44,13 +44,16 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_CCS811.git"
 
+_BASELINE_PACKING_FORMAT = "<H"
```
Comment (ladyada) [inline adafruit_ccs811.py]: you can remove this line - it uses more memory than just having <H in both spots

File: adafruit_ccs811.py [hunk 2]
```diff
@@ -156,6 +158,31 @@ def _update_data(self):
...
+        Later the same integer can be passed `set_baseline` function
+        in order to set a new baseline.
+        """
+        buf = bytearray(3)
+        buf[0] = _BASELINE
+        with self.i2c_device as i2c:
+            i2c.write_then_readinto(buf, buf, out_end=1, in_start=1)
+        return struct.unpack(_BASELINE_PACKING_FORMAT, buf[1:])[0]
+
+    def set_baseline(self, baseline_int):
```
Comment (ladyada) [inline adafruit_ccs811.py]: make this a property setter so its one variable!

**Review by ladyada — CHANGES_REQUESTED** [review]

File: adafruit_ccs811.py (hunk 2 above)
Comment (tannewt) [inline adafruit_ccs811.py]: Here is the change for what @ladyada is suggesting.
```suggestion
    @baseline.setter
    def baseline(self, baseline_int):
```

File: adafruit_ccs811.py (hunk 1 above)
Comment (anlorn) [reply adafruit_ccs811.py]: removed

File: adafruit_ccs811.py (hunk 2 above)
Comment (anlorn) [reply adafruit_ccs811.py]: updated, now this is a property

Comment (anlorn) [reply adafruit_ccs811.py]: thank you, updated

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
awesome, thank you!!!

**Pull request merged**